code/modules/mob/living/simple_animal/hostile/giant_spider/nurse.dm

```diff
@@ -158,4 +158,31 @@
 
 		else
 			busy = 0
-			stop_automated_movement = 0
\ No newline at end of file
+			stop_automated_movement = 0
+
+/mob/living/simple_animal/hostile/giant_spider/nurse/queen_spider
+	name = "spider queen"
+	desc = "Massive, dark, and very furry. This is an absolutely massive spider. Its fangs are almost as big as you!"
+	icon = 'icons/mob/giantmobs.dmi'			//Both the alien queen sprite and the queen spider sprite are 64x64, it seemed pointless to make a new file for two new states
+	icon_state = "spider_queen1"
+	icon_living = "spider_queen1"
+	icon_dead = "spider_queen_dead"
+	pixel_x = -16
+	pixel_y = -16
```

Comment:
Big hitboxes for mobs don't work well last time I tried it

It can only attack you (and you can only attack it) if you're standing next to its lower left corner.